### PR TITLE
Update iframe tests

### LINKED ISSUE #104: Make test site for iframe tests
Currently, we are targeting various domains for our iframe tests.  Ideally we would have our own test site that we can use.  Perhaps just a collection of HTML files that are served with the `serve` package?

### PR DESCRIPTION
**Pull request by mskelton**
Fixes #104

This updates our iframe tests to have our own testing page so we have better control over the tests and aren't subjected to the whims of the sites we are currently depending on.

### CODE REVIEW

File: config/global-setup.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const fs = require("fs")
+const path = require("path")
+const setup = require("jest-playwright-preset/setup")
+
+const requestListener = (html) => (_, res) => {
+  res.writeHead(200)
+  res.end(html)
+}
+
+module.exports = async (config) => {
```
Comment (mxschmitt) [inline config/global-setup.js:11]: (I think if you want, you can even write the setup and teardown in TypeScript)

**Review by mxschmitt — APPROVED** [review]
this is awesome!

File: config/global-setup.js (hunk 1 above)
Comment (mskelton) [reply config/global-setup.js:11]: Ooh, nice!

Comment (mskelton) [reply config/global-setup.js]: Updated!

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**